page.php

```diff
@@ -1,5 +1,32 @@
 <?php
 get_header();
+
+function render_submenu($post) {
```

Comment:
i do say so
because its ugly and i dont like ugly code on our github
because having the functions in functions.php is good practice and good practices' scope goes beyond this single project
because it's easy to assume most custom functions are there

dont worry, ill fix it